xds/internal/client/cds_test.go

```diff
@@ -68,6 +68,22 @@ func (s) TestValidateCluster_Failure(t *testing.T) {
 			wantUpdate: emptyUpdate,
 			wantErr:    true,
 		},
+		{
+			name: "non-supported-cluster-type",
```

Comment:
This is the same name as above. Pick a different one.

Reply:
Changed to "non-supported-cluster-type-static" and "non-supported-cluster-type-original-dst".